src/main.js

```diff
@@ -59,12 +59,12 @@ const shouldGenerate = (propType) => {
   )
 }
 
-const generateOneProp = (propType, propName) => {
-  const generate = options.generators[propType.type]
+const generateOneProp = (propType, propName, wrapInArray=true) => {
```

Comment:
`wrapInArray` may be better off with a different name but I was struggling to find one.

Reply:
`wrapInArray` might be something like `asPair` or more verbosely `asKeyValuePair` — it was always just used in the [`.fromPairs` call](https://github.com/markalfred/react-generate-props/pull/21/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR107). `wrapInArray` _is_ describing what it's doing though, so I think it's cool. This makes me think that I might want a more robust way for the primary `generateProps` call to invoke this function and get the structure that it needs back. In other words, let` generateProps` handle massaging the data into a form that it likes — don't put that responsibility on `generateOneProp`. I'll take this on in my next update 👍